feature-libs/my-account/organization/components/order-approval/list/order-approval-list.component.ts

```diff
@@ -0,0 +1,79 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import {
+  EntitiesModel,
+  RoutingService,
+  TranslationService,
+} from '@spartacus/core';
+import {
+  B2BSearchConfig,
+  OrderApproval,
+  OrderApprovalService,
+} from '@spartacus/my-account/organization/core';
+import { combineLatest, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-order-approval-list',
+  templateUrl: './order-approval-list.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OrderApprovalListComponent {
+  constructor(
+    protected routing: RoutingService,
+    protected orderApprovalService: OrderApprovalService,
+    protected translation: TranslationService
+  ) {
+    this.fetchApprovalListPage({});
```

Comment:
It shouldn't be a runtime issue, but loading data in constructor looks strange to me. I'd move it to ngOnInit. At least for consistency with other places. 
Low prio

Reply:
👍

Reply:
Updated